src/pages/dogs/[id]/index.jsx

```diff
@@ -59,12 +59,19 @@ export default function IndividualDogPage() {
 
   const { setIsEdit, isEdit, handleSubmit, reset, getValues, errors } =
     useEditDog();
+  
+  useEffect(() => {
+    if (isEdit === true) {
+      setShowFormTab(false)
+      setShowLogTab(false)
+    } else {
+      setShowFormTab(true)
+      setShowLogTab(true)
+    }
+  }, [isEdit])
 
   // Fetches information about dog if exists and sets correct tabs and filters if needed
   useEffect(() => {
-    setShowLogTab(router.query?.showLogTab);
```

Comment:
This was being used to by one of the elements on the Dog table page (`/dogs`). If you click on a tag in the Recent Log Tags section, it is supposed to navigate you to the individual dog page and automatically filter and display the log that the tag is related to. We did this by adding query params `?showLogTab=true&filteredTag=Fleas` to the url. Can you factor this into your current implementation?

showFormTab is being used for a similar purpose: after a user creates a form, the form tab should be active

Reply:
done! also, there seems to be a bug in terms of the filtering logic in that it doesn't always work when you click on a tag in the recent log tags section. i'll try to take a look at it this weekend if i get the chance!